src/shipper/ShipperChartDoughnut/ShipperChartDoughnut.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Doughnut } from 'react-chartjs-2'
+
+const ShipperChartDoughnut = ({ data }) => {
```

Comment:
All we're missing here is a `...delegated` props. This will allow us to add className/Other shenanigans to the component.

```suggestion
const ShipperChartDoughnut = ({ data, ...delegated }) => {
  return <Doughnut data={data} {...delegated}/>
```